my_list.rb

```diff
@@ -0,0 +1,20 @@
+load './my_enumerable.rb'
```

Comment:
- [ Required ] It's always considered good practice in `ruby` to import files in your project's directory using the `require_relative` statement. Please use the `require_relative` statement here instead of the `load` statement.

Reply:
The change has been updated as requested ✅